hetu-heuristic-index/pom.xml

```diff
@@ -16,6 +16,7 @@
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
         <air.check.fail-duplicate-finder>false</air.check.fail-duplicate-finder>
         <org.slf4j.version>1.7.30</org.slf4j.version>
+        <powermock.version>2.0.2</powermock.version>
```

Comment:
why we need new library?

Reply:
not a new library.. already used in HIndex... just defined the name for version and used multiple location where it was hardcoded in below dep list